engine/orchestration/src/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -2194,13 +2194,13 @@ public Network createGuestNetwork(final long networkOfferingId, final String nam
         if (vlanSpecified) {
             URI uri = BroadcastDomainType.fromString(vlanId);
             //don't allow to specify vlan tag used by physical network for dynamic vlan allocation
```

Comment:
This comment is no longer valid, we should change to:

// Allow bypass of VLAN ID check for dynamic VLAN allocation ranges

Reply:
Fixed